themes/classic/templates/checkout/_partials/steps/personal-information.tpl

```diff
@@ -32,6 +32,20 @@
       <p><small>{l s='If you sign out now, your cart will be emptied.' d='Shop.Theme.Checkout'}</small></p>
     {/if}
 
+    <div class="clearfix">
+      <form method="GET" action="{$urls.pages.order}">
+        <button
+          class="continue btn btn-primary float-xs-right"
+          name="controller"
+          type="submit"
+          value="order"
+        >
+          {l s='Continue' d='Shop.Theme.Actions'}
+        </button>
+      </form>
+
+    </div>
+
```

Comment:
One little BC problem here: when using a theme that doesn't include this block (from add-ons or other custom theme), if the user clicks on step one from another step (after step 1 is completed), then there's no way for him to go back to the next step, as this button is missing and he cannot click on the other steps. So he gets stuck and needs to refresh the page.